ros2launch/ros2launch/command/launch.py

```diff
@@ -16,7 +16,11 @@
 
 from ament_index_python.packages import get_package_prefix
 from ament_index_python.packages import PackageNotFoundError
-from argcomplete.completers import FilesCompleter
+try:
+    from argcomplete.completers import FilesCompleter
+except ImportError:
+    # argcomplete is not supported on Windows
```

Comment:
nitpick: A better comment might indicate that argcomplete is optional

```suggestion
    # argcomplete is optional
```